cmd/envbuilder/main.go

```diff
@@ -23,15 +23,11 @@ import (
 )
 
 func main() {
-	root := &cobra.Command{
-		Use: "envbuilder",
-		// Hide usage because we don't want to show the
-		// "envbuilder [command] --help" output on error.
-		SilenceUsage:  true,
-		SilenceErrors: true,
-		RunE: func(cmd *cobra.Command, args []string) error {
-			options := envbuilder.OptionsFromEnv(os.LookupEnv)
-
+	options := envbuilder.Options{}
```

Comment:
This is the Go style for zero-value instantiations.

```suggestion
	var options envbuilder.Options
```

Reply:
This is a stylistic nit

https://go.dev/ref/spec#The_zero_value
https://go.dev/tour/basics/12

```bash
$ find . -name '*.go' -exec egrep '.*:=\s*\w*\s*\{\s*\}$' -rin . {} \;
./enterprise/tailnet/htmldebug.go:64:   out := HTMLDebug{}
./enterprise/coderd/coderdenttest/coderdenttest.go:131:                 lo := LicenseOptions{}
./tailnet/node_internal_test.go:586:    d := PeerDiagnostics{}
./tailnet/node_internal_test.go:601:            d := PeerDiagnostics{}
./tailnet/node_internal_test.go:634:    d := PeerDiagnostics{}
./agent/agentssh/jetbrainstrack.go:34:  d := localForwardChannelData{}
./coderd/database/dbmem/dbmem.go:3634:  templates := templateRow{}
./coderd/devtunnel/tunnel.go:110:       cfg := Config{}
./coderd/audit/request.go:411:  d := WorkspaceBuildBaggage{}
./scripts/apitypings/main.go:526:       state := structTemplateState{}
./scripts/audittypegen/main.go:32:      g := Generator{}
./codersdk/apikey.go:188:       tokenConfig := TokenConfig{}
./site/site.go:333:     noopRW := noopResponseWriter{}
..
```

We have plenty of cases of this in the coder codebase, so it's not a firm suggestion but more of a "this is how Go encourages usage, but it's pretty-much equivalent either way"

Reply:
I would agree with you @dannykopping that `var options envbuilder.Options` is more idiomatic, but I'm mostly basing that on experience over the years. I don't think any of the references mention a preference of one over the other. There's also the Go code review comments document which also doesn't mention it in any way.

https://go.dev/wiki/CodeReviewComments